osu.Game/Overlays/BeatmapSet/Scores/ScoreTable.cs

```diff
@@ -183,7 +184,7 @@ private Drawable[] createContent(int index, ScoreInfo score)
 
                 content.Add(new OsuSpriteText
                 {
-                    Text = stat.MaxCount == null ? $"{stat.Count}" : $"{stat.Count}/{stat.MaxCount}",
+                    Text = stat.MaxCount == null ? stat.Count.ToLocalisableString(@"N0") : (LocalisableString)$"{stat.Count}/{stat.MaxCount}",
```

Comment:
the second string of this ternary makes me think we may need to introduce some kind of real formattable string, rather than splitting the string in three components (similarily to how it was done in beatmap picker)

Reply:
A formattable would be ideal in general, yes. For reasons already mentioned above (in some languages placeholders may be reordered). Although that is maybe not overly relevant in this case.